test/helpers/test-bot.js

```diff
@@ -12,8 +12,14 @@ module.exports = function TestBot (opts = {}) {
   // }
 
   let stack = Server // eslint-disable-line
-    .use(require('ssb-backlinks'))
-    .use(require('ssb-query'))
+    //.use(require('ssb-backlinks'))
+    //.use(require('ssb-query'))
+    .use(require('ssb-db2/core'))
+    .use(require('ssb-classic'))
```

Comment:
```suggestion
```
this adds box1. we don't need or want it.
this may have been why `ssb.db.create` wasn't auto encrypting for you

Reply:
I'm wrong, was thinking of ssb-box1